.gitignore

```diff
@@ -71,3 +71,5 @@ test/selenium/screenshots/*
 
 # logging
 logs/*.log*
+
+edx-ora2/
```

Comment:
This was to eliminate the build from my devstack -- will remove before final merge.